tests/src/util/playgrounds/unique.ts

```diff
@@ -0,0 +1,2486 @@
+   */
+  async burnToken(signer: TSigner, collectionId: number, tokenId: number, label?: string, amount=1n): Promise<{
+    success: boolean,
+    token: number | null
+  }> {
+    if(typeof label === 'undefined') label = `collection #${collectionId}`;
+    const burnResult = await this.helper.executeExtrinsic(
+      signer,
+      'api.tx.unique.burnItem', [collectionId, tokenId, amount],
+      true, `Unable to burn token for ${label}`,
+    );
+    const burnedTokens = this.helper.util.extractTokensFromBurnResult(burnResult, label);
+    if (burnedTokens.tokens.length > 1) throw Error('Burned multiple tokens');
+    return {success: burnedTokens.success, token: burnedTokens.tokens.length > 0 ? burnedTokens.tokens[0] : null};
+  }
+
+  /**
+   * Destroys a concrete instance of NFT on behalf of the owner
+   * 
+   * @param signer keyring of signer
+   * @param collectionId ID of collection
+   * @param fromAddressObj address on behalf of which the token will be burnt
+   * @param tokenId ID of token
+   * @param label 
+   * @param amount amount of tokens to be burned. For NFT must be set to 1n
+   * @example burnTokenFrom(aliceKeyring, 10, {Substrate: "5DyN4Y92vZCjv38fg..."}, 5, {Ethereum: "0x9F0583DbB85..."})
+   * @returns ```true``` if extrinsic success, otherwise ```false```
+   */
+  async burnTokenFrom(signer: TSigner, collectionId: number, fromAddressObj: ICrossAccountId, tokenId: number, label?: string, amount=1n): Promise<boolean> {
+    if(typeof label === 'undefined') label = `collection #${collectionId}`;
+    const burnResult = await this.helper.executeExtrinsic(
+      signer,
+      'api.tx.unique.burnFrom', [collectionId, fromAddressObj, tokenId, amount],
+      true, `Unable to burn token from for ${label}`,
+    );
+    const burnedTokens = this.helper.util.extractTokensFromBurnResult(burnResult, label);
+    return burnedTokens.success && burnedTokens.tokens.length > 0;
+  }
+
+  /**
+   * Set, change, or remove approved address to transfer the ownership of the NFT.
+   * 
+   * @param signer keyring of signer
+   * @param collectionId ID of collection
+   * @param tokenId ID of token
+   * @param toAddressObj 
+   * @param label 
+   * @param amount amount of token to be approved. For NFT must be set to 1n
+   * @returns ```true``` if extrinsic success, otherwise ```false```
+   */
+  async approveToken(signer: IKeyringPair, collectionId: number, tokenId: number, toAddressObj: ICrossAccountId, label?: string, amount=1n) {
+    if(typeof label === 'undefined') label = `collection #${collectionId}`;
+    const approveResult = await this.helper.executeExtrinsic(
+      signer, 
+      'api.tx.unique.approve', [toAddressObj, collectionId, tokenId, amount],
+      true, `Unable to approve token for ${label}`,
+    );
+
+    return this.helper.util.findCollectionInEvents(approveResult.result.events, collectionId, 'common', 'Approved', label);
+  }
+
+  /**
+   * Get amount of RFT pieces approved to transfer
```

Comment:
Not only RFT, fungible and nonfungible also uses this method. `token` instead of `RFT`